worldedit-core/src/main/java/com/boydti/fawe/object/clipboard/EmptyClipboard.kt

```diff
@@ -64,7 +64,11 @@ object EmptyClipboard : Clipboard {
         return false
     }
 
-    override fun setBiome(position: BlockVector2, biome: BiomeType): Boolean {
+    override fun setBiome(position: BlockVector3, biome: BiomeType): Boolean {
```

Comment:
I'm still surprised why we have random Kotlin classes here and there.

Reply:
I'm still interested in exploring potential uses for unsigned integers.